html_css/accessibility_auditing.md

```diff
@@ -0,0 +1,119 @@
+
+As you can see from the image above, the accessibility tree in Firefox looks sort of like a more bare-bones version of the DOM. The only information listed is the role and the name property of each element on the page. You can expand each object to see its children's role and name properties, and you can continue doing so until you reach the end of that "branch". If you right click anywhere within the accessibility tree, you can even select a "Print to JSON" option, which will open the contents of the tree as JSON in a new browser tab.
+
+The accessibility tree itself isn't too complicated, so there's not actually a lot to go over. Did you notice anything different about the element picker tool, though?
+
+![Firefox's picker tools comparison](https://user-images.githubusercontent.com/70952936/125674362-70bc34c9-73f2-46d2-bce5-877c8060327b.png)
+
+When you're in the **Inspector** tab, you can use the "element picker" tool to hover over an element on the page to see a visual representation of its box model, or you can click on an element and it will select that element in the **Inspector** tab. When you're in the **Accessibility** tab, though, this tool becomes an "accessible object picker" tool. Once you activate this tool you can hover over elements on the page to view some a11y information or you can click on an element to select it in the accessibility tree.
+
+### Chrome's Accessibility Tree
+
+There's only one way to view the accessibility tree within Chrome: open the dev tools however you normally would, then within the **Elements** tab select the **Accessibility** pane on the right side of the window (it should be in the same area as the **Styles** pane, though you may have to open the overflow menu). If you don't see this pane anywhere, you may need to enable an experimental feature within the dev tools.
+
+![Chrome's accessibility tree](https://user-images.githubusercontent.com/70952936/125674441-44095206-0508-45d0-ab2d-31ed4a2d00cb.jpg)
+
+The first thing you'll notice about the accessibility tree in Chrome is that it's laid out a little differently compared to Firefox. It still lists the role and name property of each object, but it isn't separated into two separate columns. Another difference is how you're able to view the tree itself. In Firefox, regardless of which object you select, you'll still be able to see the entire tree at once. In Chrome, you only see one branch at a time. If you clicked on the `contentinfo` object from the image above, for example, the two `generic` branches would no longer be visible.
+
+It is worth noting that you may be able to go into the Chrome dev tools settings and then into the **Experiments** section to check off the "Enable full accessibility tree view in the Elements panel" option. This will allow you to view an accessibility tree that is similar in style to Firefox's.
+
+### Firefox's Accessibility Properties
```

Comment:
I'm a little torn on having both FF and Chrome here.... In general our curriculum just assumes Chrome.

In this case, I think they're different enough that it's worth keeping the FF stuff, BUT can we list chrome first, and then group the FF and Chrome stuff in their own sections instead of alternating? Students will just want to skip to the browser they're using... i.e.:
```
### Chrome
#### Accessibility Tree
#### Accessibility Properties
#### Checks

### Firefox
#### etc....
```